TicTacToeGame/Controllers/BoardController.cs

```diff
@@ -37,19 +37,73 @@ public string Get(int id)
         }
 
         // PUT: api/Board/5
-        public void Put(int id)
+        public String Put(int id)
         {
             int y = (id % 10) - 1;
             int x = (id / 10) - 1;
             countOfTimesBoardIsCalled++;
             if (countOfTimesBoardIsCalled % 2 != 0)//player 1's turn
             {
                 board.boardcoordinates[x, y] = 'O';
+                if (ifItsAwin('O')==1)
+                {
+                    return "Player 1 wins";
+                }
             }
             else //playes 2's turn
             {
                 board.boardcoordinates[x, y] = 'X';
+                if (ifItsAwin('X') == 1)
+                {
+                    return "Player 2 wins";
+                }
+            }
+            return null;
+        }
+
+        public static int ifItsAwin(Char x)
```

Comment:
It is a good practice to follow Pascal casing for function names.

Reply:
updated and also modulated the code by differentiating controller functions and other functions by creating a functions class